.resinci.json

```diff
@@ -0,0 +1,47 @@
+{
+  "dependencies": {
+    "linux": [
+      "libudev-dev",
+      "libusb-1.0-0-dev",
+      "libyaml-dev"
```

Comment:
I'm obviously not aware (yet) of how this file is going to be used, but it seems a bit strange that Windows and macOS won't also need YAML libraries?

Reply:
Its a build-time dependency of a certain native add-on on Linux. I didn't dig into it, but maybe the module is Linux-only, or does things differently in other OSes?